src/build.py

```diff
@@ -1103,6 +1103,7 @@ def TestBare():
       runner=os.path.join(INSTALL_BIN, 'wasm.opt'),
       indir=s2wasm_out,
       fails=SPEC_KNOWN_TORTURE_FAILURES,
+      warn_only=True,  # TODO all the spec tests fail with syntax error
```

Comment:
Heads-up: spec/binary-0xc is now merged into master, and I fixed the offending WebAssembly/spec#353.